scipy/sparse/csgraph/tests/test_shortest_path.py

```diff
@@ -97,6 +97,22 @@ def check(method, directed_in):
             check(method, directed_in)
 
 
+def test_dijkstra_indices_multi_target():
+    indices = np.arange(4)
+    SP = dijkstra(directed_G,
+                  directed=False,
+                  indices=indices,
+                  multi_target=True)
+    assert_array_almost_equal(SP, np.min(undirected_SP[indices, :], axis=0))
+
+    ans = np.min(np.array(directed_SP)[indices, :], axis=0)
+    SP = dijkstra(directed_G,
+                  directed=True,
+                  indices=indices,
+                  multi_target=True)
+    assert_array_almost_equal(SP, ans)
```

Comment:
If you add the `min_indices` output proposed above, you can also check that the `argmax` of the `multi_target=False` (or maybe now `min_only=False`) result is the same as the `min_indices` of the `multi_target/min_only=True` case.

Reply:
see above comment.

Reply:
see above follow on comment... i have now fixed the tests to use this 'sources' output to check against the answers, as well as fixing the tests to use more combinations of indices for testing.